docs/source/environment.rst

```diff
@@ -0,0 +1,112 @@
+.. _Environment:
+
+Environment
+***********
+This page describes the environments in which Fab can be run.
+
+
+Requirements
+============
+The minimum dependencies for Fab to build a project are:
+
+ * Python >= 3.7
+ * Compiler
+ * Linker (or archiver, if building libraries)
+
+If your project includes C code:
+
+ * libclang
+ * python-clang
+
+
+Running Fab at The Met Office
+=============================
+
+VDI
+---
+This is currently a little awkward but will hopefully become simpler soon.
+We use a combination of *module* commands and *conda* to get Fab running at The Met Office.
+If you don't need to compile C code, you may not need conda.
+We create the conda environment as follows::
+
+    conda env create -f envs/conda/dev_env.yml
+
+Then :ref:`install fab<Install>`.
+
+.. note::
+
+    We do this in a new terminal, before any module commands.
+
+We then set up our environment *in a new terminal* as follows.
+
+For use with gfortran::
+
+    module use /data/users/lfric/modules/modulefiles.rhel7
+    module load environment/lfric/gnu
+    conda activate sci-fab
+    PYTHONPATH=~/.conda/envs/sci-fab/lib/python3.7/site-packages:$PYTHONPATH
```

Comment:
We need it to use the latest version of fparser. Otherwise we get the older version from the modules commands, which fails to parse certain files. A lot of users might not notice the difference.

I'm not happy using a mix of modules and conda, which is one reason we explored singularity.

Reply:
Clearly my testing was at fault.  I'll mark this as resolved.